### PR TITLE
Fix nmap error in travis

### PR DESCRIPTION
**Pull request by johscheuer**

### CODE REVIEW

**Comment by johscheuer** [comment]
This was a fun bug which was solved the [nsenter](https://github.com/zalando/python-nsenter/commit/b7fd78fef24c456d88130c75fe734417728e97e8) branch. Before the old `pathlib` was installed which somehow broke with Python 3.4+ funny that we didn't saw this issue earlier.

**Comment by johscheuer** [comment]
I will squash the commits once LGTM

File: tests/test_e2e.py [hunk 1]
```diff
@@ -66,15 +69,25 @@ def test__e2e__clean_setup__results_are_expected(e2e_test_case, api_client, apps
...
+        except subprocess.CalledProcessError as cpe:
+            print(cpe)
     # load contents of result and expected
     result = None
-    with open(result_file_name, "r") as stream:
-        result = yaml.safe_load(stream)
+    try:
+        with open(result_file_name, "r") as stream:
+            result = yaml.safe_load(stream)
+    except OSError:
```
Comment (maxbischoff) [inline tests/test_e2e.py:81]: In what cases do we expect this exception to appear?

**Review by maxbischoff — COMMENTED** [review]
One questions, otherwise LGTM

File: tests/test_e2e.py (hunk 1 above)
Comment (johscheuer) [reply tests/test_e2e.py:81]: When for some kind of reasons `illuminatio` can't be successfully run and won't write the file another way would be to directly fail if the command has an error.

**Review by maxbischoff — APPROVED** [review]

**Pull request merged**